### PR TITLE
Checkout improvements

### PR DESCRIPTION
**Pull request by kulczy**
image
image
image

### CODE REVIEW

File: src/Sylius/Bundle/ShopBundle/templates/shared/order/show/summary/statuses/shipments/list/state_label.html.twig [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 {% set state = hookable_metadata.context.shipment.state %}
 
-<div {{ sylius_test_html_attribute('shipment-state') }}>
-    {{  label(state) }}
-</div>
+{% if state != 'cart' %}
+    <div {{ sylius_test_html_attribute('shipment-state') }}>
+        {{  label(state) }}
```
Comment (bartek-sek) [inline src/Sylius/Bundle/ShopBundle/templates/shared/order/show/summary/statuses/shipments/list/state_label.html.twig]: Additional space

**Review by bartek-sek — DISMISSED** [review]

File: src/Sylius/Bundle/ShopBundle/templates/shared/order/show/summary/table_summary/shipping/value.html.twig [hunk 2]
```diff
@@ -0,0 +1,35 @@
+{% import "@SyliusShop/shared/macro/money.html.twig" as money %}
+
+{% set order = hookable_metadata.context.order %}
+{% set orderShippingPromotionAdjustment = constant('Sylius\\Component\\Core\\Model\\AdjustmentInterface::ORDER_SHIPPING_PROMOTION_ADJUSTMENT') %}
```
Comment (bartek-sek) [inline src/Sylius/Bundle/ShopBundle/templates/shared/order/show/summary/table_summary/shipping/value.html.twig]: Please use snake_case notation for variables names

File: src/Sylius/Bundle/ShopBundle/templates/checkout/complete.html.twig [hunk 3]
```diff
@@ -6,7 +6,9 @@
 {% set active_step = 'complete' %}
 
 {% block content %}
-    {% hook 'sylius_shop.checkout.complete' with { order, form, active_step } %}
+    <div class="mx-auto" style="max-width: var(--bs-breakpoint-lg)">
```
Comment (mpysiak) [inline src/Sylius/Bundle/ShopBundle/templates/checkout/complete.html.twig:9]: no inline styles please

File: src/Sylius/Bundle/ShopBundle/templates/shared/order/show/summary/table.html.twig [hunk 4]
```diff
@@ -1,5 +1,5 @@
-<div class="table-responsive">
-    <table class="table table-borderless align-middle" {{ sylius_test_html_attribute('order-table') }}>
+<div class="table-responsive border-bottom mb-4" style="--bs-border-style: dashed">
+    <table class="table table-borderless table-space align-middle mb-0" {{ sylius_test_html_attribute('order-table') }}>
```
Comment (mpysiak) [inline src/Sylius/Bundle/ShopBundle/templates/shared/order/show/summary/table.html.twig:2]: same here

**Review by GSadee — APPROVED** [review]

**Pull request merged**